substrate/src/lib.rs

```diff
@@ -55,9 +55,22 @@
 //! is merely a matter of the Wasm blob being changed in the chain state, which is, in principle,
 //! same as updating an account's balance.
 //!
-//! To learn more about the substrate architecture using some visuals, see [`substrate_diagram`].
+//! ### Architecture
 //!
-//! `FRAME`, Substrate's default runtime development library takes the above even further by
+//! Therefore, Substrate can be visualized as follows:
```

Comment:
I suggest to giving any further review on this file, as it is being moved in https://github.com/paritytech/polkadot-sdk/compare/kiz-frame-api...kiz-umbrella-crates?expand=1